ssh-encoding/src/encode.rs

```diff
@@ -155,7 +155,7 @@ impl Encode for [u8] {
     }
 }
 
-/// Encodes `[u8; N]` into `byte[n]` as described in [RFC4251 § 5]:
+/// Encodes `[u8; N]` into `byte[N]` as described in [RFC4251 § 5]:
```

Comment:
The RFC calls it `byte[n]`

Reply:
> `byte[n]`, where n is the number of bytes in the array.

In this case the number of bytes in the array is denoted `N`, so it would be `byte[N]`. I'm happy to remove that line from the diff, but it just irked my brain that the lowercase n was technically undefined in that sentence.

Reply:
Perhaps remove the `[u8; N]` part of the comment then?

```suggestion
/// Encodes byte array using `byte[n]` encoding as described in [RFC4251 § 5]:
```